frontend/src/scenes/persons/personsLogic.ts

```diff
@@ -18,8 +18,15 @@ interface PersonPaginatedResponse {
 
 const FILTER_ALLOWLIST: string[] = ['is_identified', 'search', 'cohort']
 
-export const personsLogic = kea<personsLogicType<PersonPaginatedResponse>>({
-    path: ['scenes', 'persons', 'personsLogic'],
+export interface PersonLogicProps {
+    cohort?: number | 'new' | 'personsModalNew'
+    syncWithUrl?: boolean
+}
+
+export const personsLogic = kea<personsLogicType<PersonLogicProps, PersonPaginatedResponse>>({
+    props: {} as PersonLogicProps,
+    key: (props) => (props.cohort ? `cohort_${props.cohort}` : props.syncWithUrl ? 'scene' : 'default'),
```

Comment:
Is there a case were we actually want `default`? If not, then maybe it should throw an error to avoid inconsistencies like the one fixed by this PR

Reply:
sure